build/configs/rtl8720e/board_metadata.txt

```diff
@@ -0,0 +1,28 @@
+###########################################################################
+#
+# Copyright 2021 Samsung Electronics All Rights Reserved.
```

Comment:
Please update the year, 2021 -> 2022.

Reply:
Thank you! I have checked and update the rest files